Tests/test_map.gd

```diff
@@ -109,7 +109,6 @@ func test_rule_3_map_gen() -> void:
 					 test_map.rooms[index_height][index_width].room_event.get_event_name(), test_map.rooms[index_height][index_width].room_position])
 
 		# Check if the two rightmost rooms of the floor are the same type or not
-		queue.pop_front()
```

Comment:
wasn't that needed to have correct size for the test ?

Reply:
Ah right you’re correct, i’ll revert it

Reply:
it's even more surprising that the test is still passing even without it